src/main/resources/conf/testmode.conf

```diff
@@ -48,7 +48,11 @@ mantis {
     }
   }
 
-  network.rpc {
-    apis = "eth,web3,net,personal,mantis,test,iele,debug,qa,checkpointing"
+  network {
+    rpc {
+      apis = "eth,web3,net,personal,mantis,test,iele,debug,qa,checkpointing"
+    }

```

Comment:
I think this should be `rpc.apis` to prevent overriding (but I'm not 100% sure. but it's shorter anyways.)